escposprinter/src/main/java/com/dantsu/escposprinter/EscPosPrinterCommands.java

```diff
@@ -446,6 +446,22 @@ public EscPosPrinterCommands feedPaper(int dots) throws EscPosConnectionExceptio
 
         return this;
     }
+    
+        /**
+     * open the cash box
+     *
+     * @return Fluent interface
+     * @throws EscPosConnectionException
+     */
+    public EscPosPrinterCommands openCashBox() throws EscPosConnectionException {
+        if (!this.printerConnection.isConnected()) {
+            return this;
+        }
+
+        this.printerConnection.write(new byte[]{0x1B, 0x70, 0x00, 0x3C, (byte) 0xFF});
```

Comment:
You can remove `(byte)`

Reply:
0xff out of byte value,so must cast to byte

Reply:
Sorry, yes you right, forget it ^^